swagger.json

```diff
@@ -1820,7 +1824,7 @@
           {
             "name": "lang",
             "in": "path",
-            "description": "Language identifier such as `'R'`",
```

Comment:
The code environment is already highlighting the data with another font family. It also encourages users to think the quotes are part of the response/code/... and need to be included. Changed that throughout the whole file.